### PR TITLE
New package py-gffutils

### PR DESCRIPTION
**Pull request by dorton21**
depends on #22551

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-gffutils/package.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    """GFF and GTF file manipulation and interconversion
+
+    gffutils is a Python package for working with and manipulating the GFF and
+    GTF format files typically used for genomic annotations. Files are loaded
+    into a sqlite3 database, allowing much more complex manipulation of
+    hierarchical features (e.g., genes, transcripts, and exons) than is
+    possible with plain-text methods alone."""
+
+    homepage = "https://github.com/daler/gffutils"
+    url      = "https://github.com/daler/gffutils/archive/refs/tags/v0.10.1.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gffutils/package.py]: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Review by haampie — APPROVED** [review]

**Comment by dorton21** [comment]
Can I get an update on this?

**Comment by haampie** [comment]
``` 
E       AssertionError: These packages have missing dependencies:
E         py-gffutils: [py-argh]
E       assert not {'py-gffutils': {'py-argh'}}
```

**Comment by haampie** [comment]
normally ci runs on a merge commit with develop, not sure what's wrong here. can you maybe rebase on develop and  push again?

**Comment by adamjstewart** [comment]
You just have to close and reopen the PR to get it to rerun with dependent packages that have already been merged into develop.

**Comment by dorton21** [comment]
Thanks

**Pull request merged**